dispatcher/dispatcher.go

```diff
@@ -340,7 +334,7 @@ func (d *IotxDispatcher) HandleTell(ctx context.Context, chainID uint32, peer pe
 	switch msgType {
 	case protogen.MsgBlockSyncReqType:
 		d.dispatchBlockSyncReq(ctx, chainID, peer, message)
-	case protogen.MsgBlockSyncDataType:
+	case protogen.MsgBlockProtoMsgType:
 		d.dispatchBlockSyncData(ctx, chainID, message)
```

Comment:
dispatchBlockCommit?

Reply:
i think it's correct to still use dispatchBlockSyncData()
this is to process old block data sent by peers from sync request, the code is different from dispatchBlockCommit () to process latest broadcast block